client/src/components/common/card/BookCard.jsx

```
@@ -31,7 +31,9 @@ export default function BookCard({ bookDetail }) {
               <div className={styles.subContent}>{author}</div>
             </div>
           </div>
-          <BookDetailItem bookId={id} initialIsLike={like}></BookDetailItem>
+          <div className={styles.bottomWrapper}>
+            <BookDetailItem bookId={id} initialIsLike={like}></BookDetailItem>
+          </div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)